src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.consortia.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.consortia.service.TenantService;
+import org.folio.pv.domain.dto.TenantCollection;
+import org.folio.consortia.repository.TenantRepository;
+import org.folio.consortia.repository.entity.Tenant;
+import org.folio.spring.data.OffsetRequest;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Log4j2
+@RequiredArgsConstructor
+public class TenantServiceImpl implements TenantService {
+  private final TenantRepository repository;
+
+  @Transactional(readOnly = true)
+  @Override
+  public TenantCollection get(Integer offset, Integer limit) {
+    var result = new TenantCollection();
+    Page<Tenant> page = repository.findAll(new OffsetRequest(offset, limit));
+    result.setTenants(page.map(TenantServiceImpl::entityToDto).getContent());
+    result.setTotalRecords(page.map(TenantServiceImpl::entityToDto).getContent().size());
```

Comment:
Does it return the quantity without the offset and limit?

Reply:
if By quantity you mean totalRecords , Yes it returns.
![image](https://user-images.githubusercontent.com/90304030/223640937-c04c83e1-bc61-42d0-a388-28c11e82b527.png)

Reply:
@SinghAdes totalRecords is calculated incorrectly, now it equals to size of return list, but it should be the size of all records in DB

Reply:
@SerhiiNosko :- it is calculating correctly , attached image is not covering all tenants.